### PR TITLE
CASSANDRA-17547

### PR DESCRIPTION
**Pull request by sharanf**
Documentation from Partition Denylist Lost in Document Migration + Minor Fixes
(https://issues.apache.org/jira/browse/CASSANDRA-17547[)]

- Recover and add denylist documentation

### CODE REVIEW

File: doc/modules/cassandra/pages/operating/denylisting_partitions.adoc [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ - Id is a simple type - INSERT INTO system_distributed.denylisted_partitions (ks_name, table_name, partition_key) VALUES ('ks','table1','1');
+ - Id is a blob        - INSERT INTO system_distributed.denylisted_partitions (ks_name, table_name, partition_key) VALUES ('ks','table1','12345f');
+ - Id has a colon      - INSERT INTO system_distributed.denylisted_partitions (ks_name, table_name, partition_key) VALUES ('ks','table1','1\:2');
+
+In the case of composite column partition keys (Key1, Key2):
+
+ - INSERT INTO system_distributed.denylisted_partitions (ks_name, table_name, partition_key) VALUES ('ks', 'table1', 'k11:k21')
+
+
+ === Special considerations
```
Comment (jmckenzie-dev) [inline doc/modules/cassandra/pages/operating/denylisting_partitions.adoc]: The leading whitespace on this section is leading to things being formatted incorrectly (at least in intellij preview; assuming that will hold true for generated pages?)

**Review by jmckenzie-dev — CHANGES_REQUESTED** [review]
Little leading whitespace trouble but once tidied up looks good.

**Comment by sharanf** [comment]
> Little leading whitespace trouble but once tidied up looks good.

Thanks for the feedback @josh-mckenzie . Have removed the ones I found - so hope this clears them

**Review by jmckenzie-dev — APPROVED** [review]
👍

**Pull request closed without merging**